.github/CONTRIBUTING.md

```diff
@@ -153,27 +121,28 @@ expect:
 
 ### Pull Request Lifecycle
 
-1. You are welcome to submit your pull request for commentary or review before
-   it is fully completed. Please prefix the title of your pull request with
-   "[WIP]" to indicate this. It's also a good idea to include specific
-   questions or items you'd like feedback on.
+1. [Fork the GitHub repository](https://help.github.com/en/articles/fork-a-repo),
+   modify the code, and [create a pull request](https://help.github.com/en/articles/creating-a-pull-request-from-a-fork).
+   You are welcome to submit your pull request for commentary or review before
+   it is fully completed by creating a [draft pull request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests)
+   and including specific questions or items you'd like feedback on.
 
-2. Once you believe your pull request is ready to be merged, you can remove the
-   "[WIP]" prefix from the title and a team member will review. Follow [the
-   checklists below](#checklists-for-contribution) to help ensure that your
-   contribution will be merged quickly.
+1. Once you believe your pull request is ready to be reviewed, ensure the
+   pull request is not a draft pull request, and a team member will review. Follow
```

Comment:
Is there a guide on how to make sure it's not a draft PR? (https://help.github.com/en/articles/changing-the-stage-of-a-pull-request maybe?)

Reply:
Can draft PRs be created outside of the GitHub Frontend? If not, I think we should still encourage the use of WIP PRs if draft PRs is not something contributors are use to or if their tooling is not capable of submitting/ creating.

Reply:
I've hopefully addressed both of these in https://github.com/terraform-providers/terraform-provider-aws/pull/9031/commits/133dcbf0b65fee71eca25588c3c91621acd1a8db 😄 Thanks for the recommendations!